### PR TITLE
Removed inactive servers

### PR DESCRIPTION
**Pull request by toohard2explain**
## Type of change

- [ ] Add new directory for a server
- [ ] Update directory of a server
- [x] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

## Further comments

All servers were unavailable for weeks

### CODE REVIEW

**Comment by ItzHoffe** [comment]
firecraft need to also be delated.

**Comment by maaaathis** [comment]
> firecraft need to also be deleted.

It's maybe a re-release? @ItzHoffe 
image

File: minecraft_servers/zyneonstudios/manifest.json [hunk 1]
```diff
@@ -1,20 +0,0 @@
...
-    "discord": {
-      "server_id": 892032464384585768,
-      "rename_to_minecraft_name": false
-    },
-    "brand": {
-      "primary": "#008FE8",
-      "background": "#0A56A5",
-      "text": "#FFFFFF"
-     }
-  }
```
Comment (maaaathis) [inline minecraft_servers/zyneonstudios/manifest.json:20]: Can you delete the whole directory for zyneonstudios?

**Review by maaaathis — CHANGES_REQUESTED** [review]

File: minecraft_servers/zyneonstudios/manifest.json (hunk 1 above)
Comment (toohard2explain) [reply minecraft_servers/zyneonstudios/manifest.json:20]: done.

Comment (maaaathis) [reply minecraft_servers/zyneonstudios/manifest.json:20]: thanks!

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**